File: problems/problems_1190/solution.go
package problem1190

import (
	"encoding/json"
	"log"
	"strings"
)

func reverseParentheses(s string) string {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var s string

	if err := json.Unmarshal([]byte(inputValues[0]), &s); err != nil {
		log.Fatal(err)
	}

	return reverseParentheses(s)
}

File: problems/problems_1190/Solution.java
package problems.problems_1190;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public String reverseParentheses(String s) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        String s = jsonStringToString(inputJsonValues[0]);
        return JSON.toJSON(reverseParentheses(s));
    }
}
